src/libraries/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -42,7 +42,7 @@ static ThreadPoolTests()
         // Tests concurrent calls to ThreadPool.SetMinThreads. Invoked from the static constructor.
         private static void ConcurrentInitializeTest()
         {
-            RemoteExecutor.Invoke(() =>
+            RemoteExecutor.Invoke((UsePortableThreadPool) =>
```

Comment:
Nit: naming convention, if this argument is actually necessary
```suggestion
            RemoteExecutor.Invoke((usePortableThreadPool) =>
```